doc/user/guide/evaluation.rst

```diff
@@ -1,2 +1,259 @@
-Interpreting and Evaluating Expressions
-=======================================
+***************************************
+Evaluation Trace Analysis Using Nushell
+***************************************
+
+Follow the official instructions to install `Nushell <https://www.nushell.sh/book/installation.html>`__.
+
+:download:`Save this file </user/guide/sequant.nu>` in the same directory
+where the trace output file is present. We will call this directory
+'working directory'.
+
+This nu script exports the functions: ``trace eval``, ``trace cache``,
+and ``trace term``.
+
+If you are using a different shell (eg. bash, zsh, etc.), launch the
+nushell from the working directory. Make sure the executable ``nu``  is
+on your path.
+
+.. code:: sh
+
+   nu
+
+Then import the functions from the script.
+
+.. code:: nu
+
+   use sequant.nu *
+
+Now we analyze :download:`the trace from this file </user/guide/trace.txt>` in steps.
```

Comment:
what expression does this trace refer to?

Reply:
Added the mpqc4 calculation output and a reference to it in the tutorial.